src/PageNavLink.tsx

```diff
@@ -0,0 +1,20 @@
+import classnames from 'classnames';
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+
+type PageNavLinkProps = {
+  to: string,
+  title: string;
```

Comment:
For consistency, use a comma instead of a semicolon to separate the properties in the `PageNavLinkProps` type definition. 😊